app/assets/javascripts/to_do_lists.coffee

```diff
@@ -9,12 +9,14 @@ class @TodoList
     showFilter()
 
   onCreate: (options) ->
-    return renderList options unless options.error
     message = detectCreateListAlertMessage()
-    showCheckAlert message
+    return showCheckAlert message if options.error
+    return appendOnCurrentPage options if $('div.list').length < options.per_page
```

Comment:
@vlasiak, Please extract method for `$('div.list').length < options.per_page` condition.